### PR TITLE
docs(tailwind): Update tailwind-css.mdx

### PR DESCRIPTION
**Pull request by ng-hai**
Add guidance to integrate with TailwindCSS.

Related shuding/nextra-docs-template#45

### CODE REVIEW

**Comment by Edwardius** [comment]
We need this!!! Otherwise tailwind doesn't work.

File: docs/pages/docs/guide/advanced/tailwind-css.mdx [hunk 1]
```diff
@@ -24,3 +24,26 @@ module.exports = {
...
+
+Next step, create a CSS file for Tailwind directives, `globals.css` for example:
+
+```css filename="globals.css"
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+```
+
+Then import into `pages/_app.{jsx,tsx}`
```
Comment (dimaMachina) [inline docs/pages/docs/guide/advanced/tailwind-css.mdx:36]:
```suggestion
Then import into `pages/_app.jsx`
```

File: docs/pages/docs/guide/advanced/tailwind-css.mdx [hunk 2]
```diff
@@ -24,3 +24,26 @@ module.exports = {
...
+
+```tsx filename="_app.tsx"
```
Comment (dimaMachina) [inline docs/pages/docs/guide/advanced/tailwind-css.mdx:38]:
```suggestion
```jsx filename="_app.jsx"
```

File: docs/pages/docs/guide/advanced/tailwind-css.mdx [hunk 3]
```diff
@@ -24,3 +24,26 @@ module.exports = {
...
+import React from "react"
+
+import "../path/to/your/globals.css"
+
+// This default export is required in a new `pages/_app.tsx` file.
```
Comment (dimaMachina) [inline docs/pages/docs/guide/advanced/tailwind-css.mdx:43]:
```suggestion
```

File: docs/pages/docs/guide/advanced/tailwind-css.mdx [hunk 4]
```diff
@@ -24,3 +24,26 @@ module.exports = {
...
+export default function App({ Component, pageProps }) {
+  return <Component {...pageProps} />
+}
+```
+
+You might need to restart dev server to see the change.
```
Comment (dimaMachina) [inline docs/pages/docs/guide/advanced/tailwind-css.mdx:49]:
```suggestion
```

File: docs/pages/docs/guide/advanced/tailwind-css.mdx [hunk 5]
```diff
@@ -24,3 +24,24 @@ module.exports = {
...
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+```
+
+Then import into `pages/_app.jsx`
+
+```jsx filename="_app.jsx"
+import React from "react"
+
```
Comment (dimaMachina) [inline docs/pages/docs/guide/advanced/tailwind-css.mdx:40]:
```suggestion
```

**Review by dimaMachina — APPROVED** [review]

**Review by dimaMachina — APPROVED** [review]
thank you 🙏

**Pull request merged**